src/simularium/rendering/GBufferPass.ts

```diff
@@ -141,93 +129,13 @@ class GBufferPass {
+            setSceneRenderPass(scene, this.pdbGbufferMaterials);
             renderer.render(scene, camera);
             //end draw pdb
             renderer.autoClear = false;
             scene.overrideMaterial = null;
         }
 
-        renderer.autoClear = true;
-
-        // 2. fill normalbuffer
-        renderPass = GbufferRenderPass.NORMAL;
-
-        const NORMAL_CLEAR = new Color(0.0, 0.0, 0.0);
-        const NORMAL_ALPHA = -1.0;
-
-        renderer.setClearColor(NORMAL_CLEAR, NORMAL_ALPHA);
-        renderer.setRenderTarget(normalBuffer);
-
-        if (DO_INSTANCED) {
-            // draw instanced things
-            this.agentPDBGroup.visible = false;
-            this.instancedMeshGroup.visible = true;
-
-            this.fibers.setRenderPass(renderPass);
-            for (let i = 0; i < this.meshTypes.length; ++i) {
-                setRenderPass(
-                    this.meshTypes[i].getMesh(),
-                    this.meshTypes[i].getShaders(),
-                    renderPass
-                );
-            }
-            renderer.render(scene, camera);
-            // end draw instanced things
-            renderer.autoClear = false;
-            scene.overrideMaterial = null;
-        }
-
-        if (DO_PDB) {
-            this.agentPDBGroup.visible = true;
-            this.instancedMeshGroup.visible = false;
-
-            setSceneRenderPass(scene, this.pdbGbufferMaterials, renderPass);
-            renderer.render(scene, camera);
-            renderer.autoClear = false;
-            scene.overrideMaterial = null;
-        }
-
-        renderer.autoClear = true;
-
-        // 3. fill positionbuffer
-        renderPass = GbufferRenderPass.POSITION;
-
-        const POSITION_CLEAR = new Color(0.0, 0.0, 0.0);
-        const POSITION_ALPHA = -1.0;
-
-        renderer.setClearColor(POSITION_CLEAR, POSITION_ALPHA);
-        renderer.setRenderTarget(positionBuffer);
-
-        if (DO_INSTANCED) {
-            // draw instanced things
-            this.agentPDBGroup.visible = false;
-            this.instancedMeshGroup.visible = true;
-
-            this.fibers.setRenderPass(renderPass);
-            for (let i = 0; i < this.meshTypes.length; ++i) {
-                setRenderPass(
-                    this.meshTypes[i].getMesh(),
-                    this.meshTypes[i].getShaders(),
-                    renderPass
-                );
-            }
-            renderer.render(scene, camera);
-            // end draw instanced things
-            renderer.autoClear = false;
-            scene.overrideMaterial = null;
-        }
-
-        if (DO_PDB) {
-            this.agentPDBGroup.visible = true;
-            this.instancedMeshGroup.visible = false;
-
-            setSceneRenderPass(scene, this.pdbGbufferMaterials, renderPass);
-            renderer.render(scene, camera);
-            renderer.autoClear = false;
-            scene.overrideMaterial = null;
-        }
-
```

Comment:
major change here: all this code is removed because we do drawing in a single pass now